qutebrowser/browser/commands.py

```diff
@@ -832,8 +832,9 @@ def paste(self, sel=False, tab=False, bg=False, window=False):
             self._open(url, tab, bg, window)
 
     @cmdutils.register(instance='command-dispatcher', scope='window',
-                       count='count')
-    def tab_focus(self, index: {'type': (int, 'last')}=None, count=None):
+                       name=['tab-focus', 'b'], count='count',
```

Comment:
I suppose I am just used to it. Anyway I will add the buffer command and leave people to do their own thing like you say.